src/word.py

```diff
@@ -1,14 +1,15 @@
 class Word:
     def __init__(self, value: str, lemma: str):
         self.value = value
-        self.lemma = lemma
+        if lemma == '_':
```

Comment:
I think it should be in the parsing part. What if one day the will change the format to "-" instead "_". We won't eant to change the Word class.
any way - make it a constant.